docs/releases.rst

```diff
@@ -16,6 +16,8 @@ Breaking Changes
 Bugfixes
 ~~~~~~~~
 
+- Fix is_dynamic_fee_transaction when gas price is zero (`#2562
```

Comment:
We don't actually manually add release notes and this change would assume it happened under a beta release that already happened (beta.3), not the next release.

You actually just need to add a brief description of the fix, such as "Fix is_dynamic_fee_transaction when gas price is zero", to a `newsfragments/2562.bugfix.rst` and during the release process this gets added to the notes correctly.

Reply:
Done